### PR TITLE
banners content updated

### LINKED ISSUE #3555: Update banners content
### Feature description

Banners content need to be updated

### Acceptance criteria

- [x] Updates content of the banner on the top on the qiskit.org page
- [x] Updates content of the banner on the announcement section on the qiskit.org page
- [x] Updates content of the banner on the announcement section on the qiskit.org/learn page
- [x] Updates content of the banner on the top on the qiskit.org/[course] page

### PR DESCRIPTION
**Pull request by aitanaviudes**
## Changes

Closes #3555

- Content of the notifications bar on the top of the qiskit.org page updated.
- Content of the banner on the announcement section of the qiskit.org page updated.
- Content of the banner on the announcement section of the qiskit.org/learn page updated.
- Content of the notifications bar on the top of the qiskit.org/course/introduction-course page updated.

### CODE REVIEW

File: pages/learn/index.vue
Comment (y4izus) [inline pages/learn/index.vue:19]: FYI --- This text says "December 1", but the title "Nov 29". I've asked if that is correct and I'm waiting for an answer

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank you @aitanaviudes ! I've some requests before merging :D

File: pages/learn/index.vue
Comment (pandasa123) [reply pages/learn/index.vue]: November 29th is the correct date!

Comment (aitanaviudes) [reply pages/learn/index.vue]: Thank you, Yaiza!

Comment (aitanaviudes) [reply pages/learn/index.vue]: https://github.com/Qiskit/qiskit.org/commit/195a3aa48ff66488057430ba2d2987cce634d442

File: pages/learn/index.vue
Comment (aitanaviudes) [reply pages/learn/index.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

File: pages/index.vue
Comment (aitanaviudes) [reply pages/index.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

File: pages/index.vue
Comment (aitanaviudes) [reply pages/index.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

File: components/LearnCourse/LearnCourse.vue
Comment (aitanaviudes) [reply components/LearnCourse/LearnCourse.vue]: https://github.com/Qiskit/qiskit.org/commit/a5570428f288577d8209e589444f9b7f6dcc1923

**Pull request merged**
